profiles/ding2/modules/ding_user/ding_user.module

```diff
@@ -872,10 +872,14 @@ function ajax_command_ding_user_authenticate($extra_data, $redirect_uri = '') {
     }
 
     foreach ($modules as $module) {
-      $command = module_invoke($module, $hook, $redirect_uri);
+      $cmd = module_invoke($module, $hook, $redirect_uri);
```

Comment:
A clarifying comment on what is going on here would be nice.

Reply:
Added somewhat long history comment about this strange behaviour.

Reply:
Thanks for the comment! It makes it a little more clear what is going on …